DevTools/HackStudio/Editor.cpp

```diff
@@ -338,3 +379,31 @@ void Editor::navigate_to_include_if_available(String path)
 
     on_open(it->value);
 }
+
+void Editor::set_execution_position(size_t line_number)
+{
+    m_execution_position = Optional<size_t>(line_number);
+    update(breakpoint_icon_rect(line_number));
+}
+
+void Editor::clear_execution_position()
+{
+    if (!m_execution_position.has_value()) {
+        return;
+    }
+    size_t previous_position = m_execution_position.value();
+    m_execution_position = {};
+    update(breakpoint_icon_rect(previous_position));
+}
+
+RefPtr<Gfx::Bitmap> Editor::breakpoint_icon_bitmap()
+{
+    static auto bitmap = Gfx::Bitmap::load_from_file("/res/icons/breakpoint.png");
+    return bitmap;
+}
+
+RefPtr<Gfx::Bitmap> Editor::current_position_icon_bitmap()
```

Comment:
This function can just return `const Gfx::Bitmap&` since it's not passing ownership of the bitmap to the caller.
It also doesn't touch any members, so it could be `static`.